examples/build.gradle

```diff
@@ -1,16 +1,68 @@
 description = 'OpenCensus Examples'
 
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "gradle.plugin.com.github.sherter.google-java-format:google-java-format-gradle-plugin:0.6"
```

Comment:
Can this plugin also be removed?

Reply:
Yes, I leave it here just because it might add productivity. But given the fact that we don't have that many codes in examples, I suppose we can just do it manually.

Another approach is to apply the plugin in root project so that we can run formatter from parent folder. But this somehow contradict purpose of making example standalone.

So I think I should just remove this plugin.

Reply:
I think it would be okay to enforce style checks from the main project, since `examples` could still be built without those plugins.  We could add that later, though.